.github/workflows/release_win_aarch64.yml

```diff
@@ -51,9 +51,9 @@ jobs:
     - name: Build ONNX wheel
       id: build_wheel
       run: |
-        $arch = 'arm64'
+        $cmake_arch = 'ARM64'
 
-        .\workflow_scripts\protobuf\build_protobuf_win.ps1 -arch $arch
+        .\workflow_scripts\protobuf\build_protobuf_win.ps1 -cmake_arch $cmake_arch
```

Comment:
Remove $cmake_arch and specify it directly here?

Reply:
done